.project-keeper.yml

```diff
@@ -7,4 +7,4 @@ sources:
 excludes:
   - "E-PK-CORE-18: Outdated content: '.github/workflows/ci-build.yml'"
 linkReplacements:
-  - "LICENSE-exasol-jdbc.txt|https://docs.exasol.com/connect_exasol/drivers/jdbc.htm"
+  - LICENSE-exasol-jdbc.txt|https://docs.exasol.com/db/latest/connect_exasol/drivers/jdbc.htm
```

Comment:
Please drop a ticket for the driver team to update the URL in their POM.

Reply:
👍